src/mars.d

```diff
@@ -1810,105 +1810,105 @@ private void setDefaultLibrary()
  */
 private void addDefaultVersionIdentifiers()
 {
-    VersionCondition.addPredefinedGlobalIdent("DigitalMars");
+    VersionCondition.addAnyGlobal("DigitalMars");
```

Comment:
It didn't convey its purpose precisely.

`addPredefinedGlobalIdent` is a function to add any identifier to the list of globally-defined identifier, without check, unlike `addGlobalIdent`. It doesn't add only 'predefined' identifier. The very term 'predefined' was weird: `D_FooBar` is not predefined, it is reserved.

And in any case, I needed to disambiguate the C++ overload and the D overload, so the C++ version kept its original name.